usagov/usagov-2021
# Usagov 1256 prioritize critical resources

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1256

## Description


added "defer" attribute to <script> tags in differnet twig files

## Type of Changes

  - Add screenshot showing what it should look like
- [ x] Other

## Testing Instructions



go to chrome dev tools > click network > click JS > refresh to see waterfall > scripts should load all in parallel after the inital DOM loads. 

<img width="542" alt="image" src="https://github.com/user-attachments/assets/a85fea82-8b92-472e-adf0-8d772ecd028f">


### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
